iceoryx_posh/test/integrationtests/service_discovery/test_roudi_findservice.cpp

```diff
@@ -40,6 +40,70 @@ TEST_F(RoudiFindService_test, OfferSingleMethodServiceSingleInstance)
     ASSERT_THAT(*instanceContainer.begin(), Eq(IdString("instance1")));
 }
 
+TEST_F(RoudiFindService_test, OfferServiceWithDefaultDescription)
+{
+    senderRuntime->offerService(iox::capro::ServiceDescription());
```

Comment:
I think this makes no sense. A default constructed ServiceDescription is invalid, see https://github.com/eclipse/iceoryx/blob/master/iceoryx_posh/source/capro/service_description.cpp line 99.

How can a service offer an invalid service? I think we should generate an error in this case and not accept an offer.
Additionally, I think I would not like to find invalid service descriptions with `findService`. Then I would get the impression that there is something substantially wrong in the system.

Reply:
created issue for same #359